appengine/queue.yaml

```diff
@@ -17,7 +17,7 @@ queue:
   target: etl-ndt-parser
   # Average rate at which to release tasks to the service.  Default is 5/sec
   # This is actually the rate at which tokens are added to the bucket.
-  rate: 1.5/s
+  rate: 0.5/s
```

Comment:
Maybe make this 1/s ?  Right now, if we have a busy day then processing will take more than 24 hours.  6 hours of processing per day is slightly worrying. 3 hours per day is a lot less worrying.